src/Symfony/Component/Form/Extension/Validator/Type/FormTypeValidatorExtension.php

```diff
@@ -67,6 +67,14 @@ public function configureOptions(OptionsResolver $resolver)
             return is_object($constraints) ? array($constraints) : (array) $constraints;
         };
 
+        $cascadeValidationNormalizer = function (Options $options, $cascadeValidation) {
+            if ($cascadeValidation) {
```

Comment:
this will trigger the deprecation only if people set the value to true. but for the people that set it to false, it should probably also trigger a deprecation since when we remove the option completely, even `false` will break their code as it creates an unknown option error. So what we did with some options is to change the default to `null` and then trigger the deorecation when `!== null`. and return `false` for null to keep the boolean logic.